pkg/schedule/schedulers/hot_region_v2_test.go

```diff
@@ -139,13 +144,15 @@ func TestHotWriteRegionScheduleWithRevertRegionsDimFirst(t *testing.T) {
 	operatorutil.CheckTransferPeer(re, ops[1], operator.OpHotRegion, 5, 2)
 	re.True(hb.searchRevertRegions[writePeer])
 	clearPendingInfluence(hb)
+	re.NoError(failpoint.Disable("github.com/tikv/pd/pkg/schedule/schedulers/setProbabilityToMovePeer"))
```

Comment:
this failpoint use is strange

Reply:
Do you have a good idea about it?

Reply:
There doesn't seem to be a good way to remove the failpoint here, in the past we used to use a uniform global variable to control the scheduling of only write peers or only write leaders, now this is calculated in real time based on configuration